my-school/src/Components/DateSelector.js

```diff
@@ -52,7 +53,7 @@ const DateSelector = (props) => {
                 <Select 
                     id="month" 
                     name="month" 
-                    defaultValue={month ? month.num : 1} 
+                    defaultValue={defaultMonth || month.num} 
```

Comment:
This confused me at first because I generally think of a default being the fallback after the `||` or, but looking at the whole file I'm pretty sure I get what's going on.  Very cool

Reply:
Ahhhh yeah, I can see that. I can always change the name from "default" to something more clear if it might create confusion! Will have to noodle on that for a bit!

Reply:
It's no biggie. I'm pretty sure the logic is solid and there's a lot of pretty sophisticated stuff going on here! 👍